CalcualtorLogic.js

```diff
@@ -18,16 +22,89 @@ class CalculatorLogic {
         this.hostElem = document.querySelector(selector);
     };
-    render() {};
+    render() {
+        const wrapperDiv = document.createElement('div');
+        wrapperDiv.classList.add("wrapper");
+        const resultDiv = document.createElement('div');
+        resultDiv.classList.add("result");
+        resultDiv.id = "result";
+        resultDiv.appendChild(
+            document.createTextNode(this.currentResult)
+        );
+        this.resultElement = resultDiv;
+        wrapperDiv.appendChild(resultDiv);
+        const row1 = this.getButtonsRow(["7", "8", "9"]);
+        const row2 = this.getButtonsRow(["4", "5", "6"]);
+        const row3 = this.getButtonsRow(["1", "2", "3"]);
+        const row4 = this.getOperatorsRow(["*", "+", "-"]);
+        wrapperDiv.appendChild(row1);
+        wrapperDiv.appendChild(row2);
+        wrapperDiv.appendChild(row3);
+        wrapperDiv.appendChild(row4);
+        const div4 = document.createElement("div");
+        const button0 = document.createElement("button");
+        button0.dataset.number = "0";
+        button0.classList.add("button");
+        const button0Text = document.createTextNode("0");
+        button0.appendChild(button0Text);
+        const periodButton = document.createElement("button");
+        periodButton.classList.add("button");
+        const periodButtonText = document.createTextNode(".");
+        periodButton.appendChild(periodButtonText);
+        const negativeValueButton = document.createElement("button");
+        negativeValueButton.classList.add("button");
+        const negativeValueButtonText = document.createTextNode("±");
+        negativeValueButton.appendChild(negativeValueButtonText);
+        div4.appendChild(button0);
+        div4.appendChild(periodButton);
+        div4.appendChild(negativeValueButton);
+        wrapperDiv.appendChild(div4);
+        this.hostElem.appendChild(wrapperDiv);
+    };
+    getButtonsRow(buttons) {
+        const div = document.createElement("div");
+        for (let button of buttons) {
+            var buttonElement = document.createElement("button");
+            buttonElement.dataset.number = button;
+            buttonElement.classList.add("button");
+            const buttonElementText = document.createTextNode(button);
+            buttonElement.appendChild(buttonElementText);
+            div.appendChild(buttonElement);
+        }
+        return div
+    }
+    getOperatorsRow(buttons) {
+        const div = document.createElement("div");
+        for (let button of buttons) {
+            var buttonElement = document.createElement("button");
+            buttonElement.dataset.operator = button;
+            buttonElement.classList.add("button");
+            const buttonElementText = document.createTextNode(button);
+            buttonElement.appendChild(buttonElementText);
+            div.appendChild(buttonElement);
+        }
+        return div
+    }
+    onButtonNumberClick(event) {
```

Comment:
Please remove everything from this method and only keep the previous console.log 
I want you read my instructions carefully.

I said to you to listen to the click events on the button numbers which you have creatd in the `getOperatorsRow`
I am expecting to see changes in that method.
leave this method a lone please